### ISSUE TITLE
Question: Using `w3` in a new project folder

### ISSUE THREAD

**Issue by kev-ngo** [Web3-API/prototype#255]
## Issue
- I want to create a new Web3API using the `w3` command a directory where I installed `@web3api/cli` in `package.json`.
- It's giving me the following error when I type `w3` in this directory:

![1](https://user-images.githubusercontent.com/69411313/113364304-ff2b1600-9307-11eb-8c07-bb9113b71f27.png)

## Steps to reproduce
- `mkdir test && cd test`
- `npm init -y`
- `npm i @web3api/cli`
- `w3`

**Comment by kev-ngo** [comment]
Note: also tried `npm i -g @web3api/cli`

**Comment by kev-ngo** [comment]
Note: also runs into this issue using `npx w3 build`.

## Steps to reproduce
1. `git clone` this monorepo
2. `yarn`
3. `yarn build`
4. `npx w3 create api assemblyscript fun`
5. `cd my-project`
6. `npx w3 build`

## Error msg
```
/home/kevin/Documents/web3api/monorepo/fun/node_modules/gluegun/build/index.js:13
    throw up;
    ^

Error: ENOENT: no such file or directory, scandir '/home/kevin/Documents/web3api/monorepo/fun/node_modules/@web3api/cli/build/lib/intl/../../../lang/'
    at Object.readdirSync (fs.js:1021:3)
    at getLocaleData (/home/kevin/Documents/web3api/monorepo/fun/node_modules/@web3api/cli/build/lib/intl/utils.js:43:10)
    at Object.getIntl (/home/kevin/Documents/web3api/monorepo/fun/node_modules/@web3api/cli/build/lib/intl/utils.js:33:22)
    at Object.<anonymous> (/home/kevin/Documents/web3api/monorepo/fun/node_modules/@web3api/cli/build/lib/intl/types.js:7:20)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18) {
  errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: '/home/kevin/Documents/web3api/monorepo/fun/node_modules/@web3api/cli/build/lib/intl/../../../lang/'
```

**Comment by dOrgJelli** [comment]
Fixed in release `0.0.1-prealpha.11`
